### PR TITLE
#1373 Footprint family to a new package

### LINKED ISSUE #1373: Move all Footprint classes into a separate package
It's better to move all `Footprint` classes into a separate package.
For example, we can give the name `org.eolang.maven.footprint`

### PR DESCRIPTION
**Pull request by mximp**
closes #1373

### CODE REVIEW

**Comment by mximp** [comment]
@volodya-lombrozo please check

File: eo-maven-plugin/src/main/java/org/eolang/maven/footprint/package-info.java [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NON-INFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ */
+/**
+ * EO program footprint.
```
Comment (volodya-lombrozo) [inline eo-maven-plugin/src/main/java/org/eolang/maven/footprint/package-info.java:25]: Please, provide more human readable description. I remember that you have already tried to explain me what Footprint is, but I forgot.  It definitely will be useful for future newcomers (and to be honest, for ourself too).

**Review by volodya-lombrozo — CHANGES_REQUESTED** [review]

File: eo-maven-plugin/src/main/java/org/eolang/maven/footprint/package-info.java (hunk 1 above)
Comment (mximp) [reply eo-maven-plugin/src/main/java/org/eolang/maven/footprint/package-info.java:25]: fixed

**Comment by mximp** [comment]
@volodya-lombrozo please check

**Review by volodya-lombrozo — APPROVED** [review]
Looks good to me. @yegor256 please, merge

**Comment by mximp** [comment]
@yegor256 reminder

**Comment by yegor256** [comment]
@rultor merge

**Pull request merged**